python/cudf_polars/cudf_polars/experimental/shuffle.py

```diff
@@ -39,13 +39,17 @@ class RMPFIntegration:  # pragma: no cover
     @staticmethod
     def insert_partition(
         df: DataFrame,
-        on: Sequence[str],
+        partition_id: int,  # Not currently used
         partition_count: int,
         shuffler: Any,
+        options: dict[str, Any],
```

Comment:
So this can maybe be


```suggestion
        options: InertPartitionOptions,
```

with this defined somewhere (probably in this file)

```python
class TypedDict:
    on: Sequnce[str]

```

Reply:
Did you mean to do this in `examples/dask.py`? I was thinking that we wanted to keep the general protocol open to anything?